releasenotes/notes/sbplx_optimizer-a27361fc4ac8de8c.yaml

```diff
@@ -0,0 +1,8 @@
+---
+features:
+  - |
+    Support for SBPLX optimizer from NLopt library has been added.
+    SBPLX is a local gradient-free optimizer based on Nelder-Mead and
+    is expected to show better convergence behavior.
+    Further information about this and all other optimizers can be found
+    `in the API ref. here <https://qiskit-community.github.io/qiskit-algorithms/apidocs/qiskit_algorithms.optimizers.html>`__.
```

Comment:
```suggestion
    Support for :class:`.SBPLX` optimizer from NLopt library has been added.
    SBPLX is a local gradient-free optimizer based on Nelder-Mead and
    is expected to show better convergence behavior.
    Further information about this can be found in the API ref for the
    :mod:`~qiskit_algorithms.optimizers`.
```